recipe/meta.yaml

```diff
@@ -1,29 +1,30 @@
-{% set version = "1.2.4" %}
+{% set version = "1.2.5" %}
 
 package:
   name: pytoolconfig
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pytoolconfig/pytoolconfig-{{ version }}.tar.gz
-  sha256: 5e1a246f77970ddb5f3d8d06fbf162424b8a1adfc22c2eb51826b383bf293d1e
+  url: https://github.com/bagel897/pytoolconfig/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 077daa343e0b865f30eadbd2a2fc778983a526311dd44f8ae31d17ba5bc1b126
```

Comment:
I think the `pypi` link is ok, conda-forge is still using it https://github.com/conda-forge/pytoolconfig-feedstock/blob/fd1770696d7ce6eeffa63d4ac54077a32f850e42/recipe/meta.yaml#L8-L9. Or is something not working?

Reply:
When I first started this PR, the developers for pytoolconfig, hadn't updated their PyPi webpage yet. Considering this, I feel it is more efficient to utilize the GitHub link as it seems that the developers post their changes and updates to GitHub before updating on Pypi.

Reply:
They publish new releases as well https://pypi.org/project/pytoolconfig/#history

Reply:
Oh I see! After taking a look and comparing the dates. I do maintain that the GitHub URL is a bit more reliable only because Pypi published their release two days ago compared to this most recent version being posted on Github approx 3 weeks ago.